themes/10up-theme/assets/css/frontend/base/wordpress.css

```diff
@@ -76,9 +77,12 @@
 	* classes, please see:
 	*
 	* https://tailwindcss.com/docs/typography-plugin
+	*
+	* To disable the richtext styling remove the `prose` class.
 */
 .block-editor-block-list__layout,
 .entry-content {
```

Comment:
I'm not sure we want to associate prose implicitly with `entry-content`. While it makes sense from a convention standpoint (entry content shouldn't be used to wrap complex layouts) so I think we can let it ride for now, but part of me thinks its better/easier (and more the "Tailwind way") to not do this, since we control all instances of where `entry-content` appears (vs say, the `block-editor-block-list__layout` class which comes from Gutenberg). 

@dainemawer @Firestorm980 What do you two think on this? I think we could possibly let it go in our MVP and just see if we/anyone dislikes it later. I just wanted to make sure we discussed it.

Reply:
Yeah, I think on this one let's see what happens when engineers pull this down and begin work. This will definitely require some tweaking here and there! Resolving this comment for now, but thanks for bringing it up @iansvo

Reply:
@iansvo @dainemawer I'm a 100% for removing prose if that's what you're suggesting. I think it may cause more headaches and confusion than it's worth